packages/statemanager/tests/ethersStateManager.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { Address, bigIntToBuffer, bigIntToHex, bufferToHex, setLengthLeft } from '@ethereumjs/util'
+import { CloudflareProvider, JsonRpcProvider } from '@ethersproject/providers'
+import * as tape from 'tape'
+import { blockFromRpc } from '@ethereumjs/block/dist/from-rpc'
+import { VM } from '@ethereumjs/vm'
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import { FeeMarketEIP1559Transaction } from '@ethereumjs/tx'
+
+import { EthersStateManager } from '../src/ethersStateManager'
+
+const provider = new CloudflareProvider()
```

Comment:
Hmm, this is doing real requests on the Ethers standard cloudflare account on our CI, is this correct? Not sure if this is a good idea, Richard is always concerned about over-use.

Reply:
Oh really? I thought this was the public cloudflare gateway? These tests are only proof on concept to see that it works anyway. I would switch to mocked responses in a final merged version of we go with this approach.

Reply:
Or to phrase it differently: we generally should make sure that we do not leech on public Ethers API endpoints.

Reply:
[We're good] (https://github.com/ethers-io/ethers.js/blob/master/packages/providers/src.ts/cloudflare-provider.ts#L23). I looked at the code behind the cloudflare provider and it's calling the main public gateway and not associated with an ethers api key.

Reply:
👍